<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankListClass</class>
 <widget class="QWidget" name="RankListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>960</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1012</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RankList</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Agency FB&quot;;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>官方榜</string>
   </property>
  </widget>
  <widget class="QLabel" name="soar_label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>70</y>
     <width>215</width>
     <height>213</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>213</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 italic 16pt &quot;Bodoni MT Black&quot;;
background-color: #3c76a8;
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="soar_list" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>697</width>
     <height>212</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>697</width>
     <height>212</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="soar_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>查看全部</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/向右.png</normaloff>../../res/向右.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="new_song_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102, 102, 102);
font: 11pt &quot;Agency FB&quot;;
border-radius: 12px;
QPushButton#new_song_btn:hover{
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>查看全部</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../res/向右.png</normaloff>../../res/向右.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="new_song_label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>350</y>
     <width>215</width>
     <height>213</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>213</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 italic 16pt &quot;Bodoni MT Black&quot;;
background-color: #45a07e;
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="new_song_list" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>697</width>
     <height>212</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>697</width>
     <height>212</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
